src/site/content/en/blog/fast-playback-with-preload/index.md

```diff
@@ -163,6 +163,7 @@ applied in Chrome:
 
 - When [Data Saver][datasaver] is enabled, Chrome forces the `preload` value to
   `none`.
+- In Android 4.3, Chrome forces the `preload` value to `none` due to an [android-bug].
```

Comment:
Ir should be "Android bug" not "android-bug".

Reply:
Fixed.